src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -934,16 +917,16 @@ else if (CoreUtilities.toLowerCase(object.element).contains(contLow)) {
         // @returns ElementTag(Boolean)
         // @group element checking
         // @description
-        // Returns whether the element contains all of the specified strings, case sensitive.
+        // Returns whether the element contains all the specified strings, case sensitive.

```

Comment:
again, removing `of` is wrong

Reply:
Reverted & changed `strings` to `elements` here as well